### PR TITLE
docs: clarify what fixes are merged to LTS versions

### PR DESCRIPTION
**Pull request by petebacondarwin**
For ease of review - here is the changed section:
https://pr38788-064ce3a.ngbuilds.io/guide/releases#support-policy-and-schedule

### CODE REVIEW

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -107,6 +107,13 @@ Version | Status | Released     | Active Ends  | LTS Ends
 
 Angular versions ^4.0.0, ^5.0.0, ^6.0.0 and ^7.0.0 are no longer under support.
 
+### LTS fixes
+
+As a general rule, a fix is considered for an LTS version if it fixes one of:
```
Comment (aikidave) [inline aio/content/guide/releases.md:112]: Minor nit: Find a different word than "fixes"? Maybe "addresses" or "resolves"?

**Review by aikithoughts — APPROVED** [review]

File: aio/content/guide/releases.md (hunk 1 above)
Comment (petebacondarwin) [reply aio/content/guide/releases.md:112]: "Resolves" works for me.

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Review by aikithoughts — APPROVED** [review]

File: aio/content/guide/releases.md [hunk 2]
```diff
@@ -107,6 +107,13 @@ Version | Status | Released     | Active Ends  | LTS Ends
 
 Angular versions ^4.0.0, ^5.0.0, ^6.0.0 and ^7.0.0 are no longer under support.
 
+### LTS fixes
+
+As a general rule, a fix is considered for an LTS version if it resolves one of:
```
Comment (Splaktar) [inline aio/content/guide/releases.md:112]: > As a general rule,

This is added here to give us some flexibility? Is that intentional?

File: packages/compiler-cli/test/ngtsc/template_mapping_spec.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ import {getMappedSegments, SegmentMapping} from './sourcemap_utils';
 
 const testFiles = loadStandardTestFiles();
 
-runInEachFileSystem((os) => {
+runInEachFileSystem.native((os) => {
```
Comment (Splaktar) [inline packages/compiler-cli/test/ngtsc/template_mapping_spec.ts:21]: Was this change intentionally part of this commit? Is it related to LTS messaging?

Comment (petebacondarwin) [reply packages/compiler-cli/test/ngtsc/template_mapping_spec.ts:21]: Oops!

File: aio/content/guide/releases.md (hunk 2 above)
Comment (petebacondarwin) [reply aio/content/guide/releases.md:112]: Yes, it was intentional. Did you prefer it not to be there?

**Review by mgechev — APPROVED** [review]

File: aio/content/guide/releases.md (hunk 2 above)
Comment (Splaktar) [reply aio/content/guide/releases.md:112]: I just wanted to make sure that it was there for a reason and that it wasn't unintentionally making things more flexible (and up to interpretation) than desired.

**Review by Splaktar — APPROVED** [review]
LGTM

**Pull request closed without merging**